src/components/MessageBanner/MessageBanner.ts

```diff
@@ -144,7 +144,7 @@ namespace fabric {
     /**
      * hides banner when close button is clicked
      */
-    private _hideBanner(): void {
+    public _hide(): void {
```

Comment:
Please remove the underscore (`hide()` not `_hide()`) and move this code under the `show()` function.